vmdb/db/migrate/20150414204018_create_container_node_identities.rb

```diff
@@ -0,0 +1,16 @@
+class CreateContainerNodeIdentities < ActiveRecord::Migration
+  def change
+    add_column :operating_systems, :container_node_id, :bigint
```

Comment:
@simon3z Is this still needed?  I thought with moving to `ComputerSystem`, this relationship would be gone, right?